django/utils/http.py

```diff
@@ -144,7 +144,14 @@ def parse_http_date(date):
     try:
         year = int(m.group('year'))
         if year < 100:
-            if year < 70:
+            # RFC 850 does not mention this, but in RFC 7231 (and there's something similar in RFC 2822),
```

Comment:
I don't think repeating the comment from the ticket makes sense. RFC 7231 section 7.1.1.1 contains the same information, and is already referenced in the doc-string.

Reply:
I removed the comments.
Not 100% sure if I understand your note about #23919 - that ticket doesn't seem to mention emails.Util. Could you elaborate? Thanks!